src/mol-plugin-state/manager/structure/component.ts

```diff
@@ -480,6 +480,7 @@ namespace StructureComponentManager {
         ),
         visualQuality: PD.Select('auto', VisualQualityOptions, { description: 'Control the visual/rendering quality of representations' }),
         ignoreLight: PD.Boolean(false, { description: 'Ignore light for stylized rendering of representations' }),
+        celShaded: PD.Boolean(false, { description: 'Cell Shading light for stylized rendering of representations' }),
```

Comment:
that doesn't work, do you really need this here?

Reply:
I removed it